dasharo-compatibility/dasharo-tools-suite.robot

```diff
@@ -0,0 +1,150 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../variables.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Enable Network Boot
+...                     AND
+...                     Disable Secure Boot
+...                     AND
+...                     Upload And Mount DTS Flash ISO
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+DTS001.001 Booting DTS from USB works correctly
+    [Documentation]    This test aims to verify that DTS is properly booting
+    ...    from USB.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS001.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS001.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    USB_emulated
```

Comment:
This leads to `PiKVM Composite KVM Device` which means we hardcode in test that the test would only work with PiKVM. Does it make sense? It should just as well work in QEMU (as e.g. ESP scanning does already), or on another HW.

We can autimatically detect if there is a PiKVM, or we are in QEMU, or in other situation, and apply appropriete path. This selection could be hidden in this `    Boot Dasharo Tools Suite    USB_emulated` kwd